src/views/Viewer.vue

```diff
@@ -26,7 +26,7 @@
 		v-if="initiated || currentFile.modal"
 		id="viewer"
 		:class="{'icon-loading': !currentFile.loaded && !currentFile.failed}"
-		:clear-view-delay="isTesting ? -1 : 5000 /* prevent cypress timeouts */"
+		:clear-view-delay="isTesting ? -1 : isMobile ? -1 : 5000" /* prevent cypress timeouts and disable on mobile */
```

Comment:
```suggestion
		:clear-view-delay="isTesting ? -1 : isMobile ? -1 : 5000 /* prevent cypress timeouts and disable on mobile */" 
```

Reply:
Ah thanks! so it should be included in the "" :)

Reply:
`/* */` is a js comment which needs to be inside of the property ;)

Reply:
okay, didn't know that :)